get-started.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Getting Started
+summary: Quick list of instructions and hints to get started with NServiceBus and the Particular Service Platform
+reviewed: 2016-12-05
+---
+
+There are a variety of starting points for NServiceBus and the platform. The path taken will depend on an individuals specific experience and preferences for learning.
+
+
+## High Level Content
+
+There are many articles that explain concepts and technologies used in the platform. Some good articles when getting started are:
+
+ * [Particular Service Platform Overview](/platform/): Provides an overview of the various parts of Platform; [NServiceBus](/nservicebus/), [ServiceControl](/servicecontrol/), [ServiceInsight](/serviceinsight/) and [ServicePulse](/servicepulse/).
+ * [Architectural Principles](/nservicebus/architecture/principles.md)
+ * [Concepts Overview](/nservicebus/concept-overview.md): A high level overview of the concepts, features and vernacular of NServiceBus.
+
+
+## Tutorials
+
+There is a [Messaging Basics Tutorial](/tutorials/nservicebus-101/) that gives a guided walk though of building a NServiceBus system and introduces the basic concepts of messaging. The tutorial also has a downloadable and runnable solution to illustrate the finished product.
```

Comment:
How about ... The [Messaging Basics Tutorial](/tutorials/nservicebus-101/) gives a guided walk though of building a NServiceBus system and introduces the basic concepts of messaging.